src/App.tsx

```diff
@@ -15,48 +16,140 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SortByAlphabet = 'alphabet',
+  SortByLength = 'length',
+  DefaultValue = '',
+}
+
+interface SortingOptions {
+  sortType: SortType,
+  isReversed: boolean,
+}
+
+function getSortedGoodsList(
+  goods: string[],
+  { sortType, isReversed }: SortingOptions,
```

Comment:
Why do you pass the second and third arguments as one object?
What is the point in this?

Reply:
Today we had a conversation with Anna about this. In case when we are destructuring them if we would have situation when there would be much more sorting options, we wouldn't care about order. If we provide them as arguments, we should be confident that we receive them in the same order that we provided.
@roman-mirzoian